### PR TITLE
Update minio package to use RELEASE.2018-04-04T05-20-54Z

### PR DESCRIPTION
**Pull request by krisis**

### CODE REVIEW

**Comment by krisis** [comment]
@ryadav88 Could you PTAL at this PR? I have tested this change locally using universe-server docker image `20180405T004858Z-pull-1774-7bb82697de`. It works as expected.

File: repo/packages/M/minio/11/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "packagingVersion": "3.0",
```
Comment (ryadav88) [inline repo/packages/M/minio/11/package.json:2]: change to 4.0

File: repo/packages/M/minio/11/package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "maintainer": "dev@minio.io",
+  "website": "https://www.minio.io",
+  "framework": false,
+  "description": "Minio is an object storage server for cloud native applications. To deploy at scale, use DC/OS to orchestrate Minio containers one per tenant.",
+  "tags": [
+    "data",
+    "storage",
+    "filesystem"
+  ],
+  "preInstallNotes": "This is a community package. Community packages are unverified and unreviewed content from the community. Launch as many Minio instances as required referring to the documentation here https://github.com/dcos/examples/blob/master/minio/1.9/README.md",
```
Comment (ryadav88) [inline repo/packages/M/minio/11/package.json:16]: You may remove this line: `This is a community package. Community packages are unverified and unreviewed content from the community.` The section `Community Packages` in Universe says that now

Comment (ryadav88) [reply repo/packages/M/minio/11/package.json:16]: https://github.com/dcos/examples/blob/master/minio/1.9/README.md

Add an updated 1.11 example

**Review by ryadav88 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**